README.md

```diff
@@ -377,9 +378,48 @@ Do the same for each project `project2.conf`, `project3.conf`,...
 7 - Create your project Databases. Right now you have to do it manually by entering your DB container, until we automate it soon.
 
 
+<a name="Speed-MacOS"></a>
+## Improve speed on MacOS
+
+Sharing code into Docker containers with osxfs have very poor performance compared to Linux. You can get around the issue you have two options:
+ - Share files with NFS (simpler, no duplication of files)
+ - Sync files in a native volume (potentially better performances)
```

Comment:
ok, you right it will minimize support time.. I will check with docker-sync owner what he think is the best/simpler configuration for docker-sync with laradock.